src/v2/ion/i18nTransformer.js

```diff
@@ -83,15 +94,21 @@ const I18N_OVERRIDE_MAPPINGS = {
   'challenge-authenticator.phone.credentials.passcode': 'mfa.challenge.enterCode.placeholder',
   'challenge-authenticator.security_question.credentials.answer': 'mfa.challenge.answer.placeholder',
   'challenge-authenticator.app.credentials.totp': 'oie.okta_verify.totp.enterCodeText',
+
+  'enroll-profile.userProfile.lastName': 'oie.user.profile.lastname',
+  'enroll-profile.userProfile.firstName': 'oie.user.profile.firstname',
+  'enroll-profile.userProfile.email': 'oie.user.profile.primary.email'
+
 };
 
 const getI18nKey = (i18nPath) => {
   let i18nKey;
 
   // we can add mapping to `I18N_OVERRIDE_MAPPINGS` for all
   // security question. It's just a bit tedious hence use following shortcut.
-  if (i18nPath.indexOf(SECURITY_QUESTION_PREFIX) === 0) {
-    const securityQuestionValue = i18nPath.replace(SECURITY_QUESTION_PREFIX, '');
+  // Applies for enrollment, when choose a security questions has option to select from.
+  if (i18nPath.indexOf(SECURITY_QUESTION_SUBSTRING) > -1) {
```

Comment:
what else? Is it `challenge-authenticator.security_question.credentials.questionKey.`?
I might prefer to be explicit like

```
i18nPath.indexOf('challenge-authenticator.security_question.credentials.questionKey.') === 0 ||
i18nPath.indexOf('enroll-authenticator.security_question.credentials.questionKey.') === 0
```

Reply:
the other was challenge-authenticator and select-authenticator-authenticate also i guess.. There could be more in future ? 

`i18nPath.indexOf(SECURITY_QUESTION_SUBSTRING) > -1` does this expose any leaks ?

Also i wanted to ask if `mutable` : `false` why don't update ? Because these set of security questions are not mutable..

Reply:
how come `select-authenticator-authenticate` has SQ question key?

Reply:
yeah it's not.. I can make it specific for just these two views.